CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [1.1.0] - 2021-10-10
+
+### Added
+- SPIRE images are now published to GitHub Container Registry. They will continue to be published to Google Container Registry for the next release (#2576,#2580)
+- SPIRE Server now implements the TrustDomain API, which allows callers to manage dynamic federation relationships (#2497,#2524,#2532,#2533,#2535,#2536,#2541,#2544,#2545,#2546,#2547,#2549,#2550,#2557,#2559,#2560,#2561,#2562,#2563,#2564,#2565,#2566,#2568,#2571,#2572,#2574,#2575).
+- The TPM DevID Node Attestor emits a new `issuer:cn` selector with the common name of the issuing certificate (#2581)
+- The K8s Bundle Notifier plugin now supports pushing the bundle to resources in multiple clusters (#2531)
+- The SVIDStore plugin type has been introduced to enable, amongst other things, agentless workload scenarios (#2176,#2483)
+- A built-in AWS Secrets Manager SVIDStore plugin has been introduced, which can push workload SVIDs into AWS secrets (#2542)
+- The agent and entry list commands in the CLI gained additional filtering capabilities (#2478,#2479)
+
+### Changed
+- With the GA release of GCP CAS, the UpstreamAuthority plugin now needs to know which pool the CA belongs to. If not configured, it will do a pessimistic scan of all pools to locate the correct CA. This scan will be removed in a future release (#2569)
```

Comment:
I think this might justify a line in "added" new configurable for ...

I know it's a duplication, but given that "added" is the first section and often the one people are most excited to read, it feels to me like new configurables introduced in favor of old ones or whatever should go there too (if anything to just call attention to them)

Reply:
👍